packages/fluent-theme/src/styles/index.ts

```diff
@@ -0,0 +1,11 @@
+import { useMemo } from 'react';
+import useStlyeToEmotionObject from 'botframework-webchat-component/lib/hooks/internal/useStyleToEmotionObject';
+
+export function useStyles<T extends Record<`webchat__${string}`, any>>(styles: T): Record<keyof T, string> {
```

Comment:
Consider...

```suggestion
export function useStyles<T extends Record<`webchat-fluent__${string}`, any>>(styles: T): Record<keyof T, string> {
```

Reply:
Sorry, I thought we already agreed on the `webchat__` prefix. Made necessary changes.